<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Product Information</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>product id</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="productIdInput"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>product name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="productNameInput"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>quantity</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="quantityInput"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>unit price</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLineEdit" name="unitPriceInput"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>vendor id</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>category</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QComboBox" name="categorycombo"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>total price</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLineEdit" name="totalPriceInput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QComboBox" name="vendorCombo"/>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="tableWidget"/>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAddProduct">
        <property name="text">
         <string>Add Products</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdateProduct">
        <property name="text">
         <string>Update Products</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteProduct">
        <property name="text">
         <string>Delete Product</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCleanForm">
        <property name="text">
         <string>Clean Form</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProduct_Panel">
    <property name="title">
     <string>Product Panel</string>
    </property>
   </widget>
   <addaction name="menuProduct_Panel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
